### DISCUSSION TITLE
convert DB into REST API instead of spreadsheet [Apr 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1989] [Discord Threads]
> 🙋‍♂️ **manikumarnune123**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### spreadsheet is just a GUI  
#  
> 👨‍🚀 **wingkwong**  
#### once you hv a project connecting to the db, u can still use REST API without using the UI  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969147423207329793)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
